<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditGammaDialog</class>
 <widget class="QDialog" name="EditGammaDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Gamma DES</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_top">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 170, 255);
color: rgb(238, 238, 238)
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_titleApp">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Edit Gamma</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Nuclide</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nuclide_lineEdit">
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Initial level (keV)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="initial_level_lineEdit">
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Final level (keV)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="final_level_lineEdit">
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Energy γ (keV)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="energy_g_lineEdit">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Intensity γ (%)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLineEdit" name="intensity_g_lineEdit">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Total internal conversion</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QLineEdit" name="total_internal_conversion_lineEdit">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Multipolarity</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QLineEdit" name="multipolarity_lineEdit">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Conversion Coefficient Electron Probability</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <spacer name="horizontalSpacer_20">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_27">
             <property name="text">
              <string>Subshell</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="subshell_cc_comboBox">
             <item>
              <property name="text">
               <string>K</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>L1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>L2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>L3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>M1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>M2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>M3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>M4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>M5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>N1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>N2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>N3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>N4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>N5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>N6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>N7</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>O1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>O2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>O3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>O4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>O5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>O6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>O7</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>P1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>P2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>P3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>P4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>P5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Q1</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_28">
             <property name="text">
              <string>Intensity</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="intensity_cc_lineEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="addUpdate_subshell_pushButton">
             <property name="text">
              <string>Add / Update</string>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/Images/update.png</normaloff>:/Images/update.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_19">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer_15">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>37</width>
               <height>58</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QTableWidget" name="subshell_cc_tableWidget">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Subshell</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Intensity</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_16">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>37</width>
               <height>58</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="cancel_pushButton">
          <property name="text">
           <string>Cancel</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/Images/cancel.png</normaloff>:/Images/cancel.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>388</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="update_pushButton">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Update</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/Images/downloads.png</normaloff>:/Images/downloads.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>energy_g_lineEdit</tabstop>
  <tabstop>intensity_g_lineEdit</tabstop>
  <tabstop>total_internal_conversion_lineEdit</tabstop>
  <tabstop>multipolarity_lineEdit</tabstop>
  <tabstop>subshell_cc_comboBox</tabstop>
  <tabstop>intensity_cc_lineEdit</tabstop>
  <tabstop>addUpdate_subshell_pushButton</tabstop>
  <tabstop>subshell_cc_tableWidget</tabstop>
  <tabstop>update_pushButton</tabstop>
  <tabstop>cancel_pushButton</tabstop>
  <tabstop>initial_level_lineEdit</tabstop>
  <tabstop>nuclide_lineEdit</tabstop>
  <tabstop>final_level_lineEdit</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
